### ISSUE TITLE
Laravel comparability

### ISSUE THREAD

**Issue by starba3** [dcodeIO/bcrypt.js#83]
This is not a real issue, more than a question of weather if this library is compitable with laravel's built in bcrypt.
Also worth noting: laravel bcrypt uses 10 rounds.

**Comment by dcodeIO** [comment]
Unless either laravel's implementation or bcrypt.js does something wrong, yes, hashes between different libraries should be compatible.

**Comment by gersonmontenegro** [comment]
Yes, it is. I compared the hash provided by passport authentication in Laravel 5.6 using this library in a Ionic app and works like a charm like this:

`bcrypt.compareSync('yourPassword', 'yourBcryptHash')`

It return true, or false.

**Comment by rosnaib11** [comment]
I'm using laravel 5.7, with a default authentication. Yes it works,
